exe/CMakeLists.txt

```diff
@@ -9,19 +9,19 @@ add_executable(pxp-agent main.cc)
 target_link_libraries(pxp-agent libpxp-agent)
 install(TARGETS pxp-agent DESTINATION bin)
 
-set(TASK_WRAPPER_LIBS ${Boost_LIBRARIES} ${LEATHERMAN_LIBRARIES})
+set(EXECUTION_WRAPPER_LIBS ${Boost_LIBRARIES} ${LEATHERMAN_LIBRARIES})
 if (CMAKE_SYSTEM_NAME MATCHES "AIX")
     find_package(Threads)
-    list(APPEND TASK_WRAPPER_LIBS ${CMAKE_THREAD_LIBS_INIT})
+    list(APPEND EXECUTION_WRAPPER_LIBS ${CMAKE_THREAD_LIBS_INIT})
 endif()
 
-add_executable(task_wrapper task_wrapper.cc)
-target_link_libraries(task_wrapper ${TASK_WRAPPER_LIBS})
-install(TARGETS task_wrapper DESTINATION bin)
+add_executable(execution_wrapper execution_wrapper.cc)
+target_link_libraries(execution_wrapper ${EXECUTION_WRAPPER_LIBS})
+install(TARGETS execution_wrapper DESTINATION bin)
```

Comment:
There's a small chance someone has directly used the task_wrapper executable - I remember some conversations related to using it for some invocation logic, maybe with Choria. We should call out the rename in release notes when we introduce this work.